# MotorLowArray.msg - 26x低速数据数组
# 电机低速信息数组
MotorLow[] motors       # 电机低速信息数组